cli/fs_util.rs

```diff
@@ -90,17 +90,26 @@ pub fn resolve_from_cwd(path: &Path) -> Result<PathBuf, AnyError> {
 
 /// Checks if the path has extension Deno supports.
 pub fn is_supported_ext(path: &Path) -> bool {
-  let lowercase_ext = path
-    .extension()
-    .and_then(|e| e.to_str())
-    .map(|e| e.to_lowercase());
-  if let Some(ext) = lowercase_ext {
-    ext == "ts" || ext == "tsx" || ext == "js" || ext == "jsx" || ext == "mjs"
+  if let Some(ext) = get_extension(path) {
+    ext == "ts"
+      || ext == "tsx"
+      || ext == "js"
+      || ext == "jsx"
+      || ext == "mjs"
+      || ext == "md"
```

Comment:
```suggestion
    matches!(ext.as_str(), "ts" | "tsx" | "js" | "jsx" | "mjs" | "md");
```
small refactor to make this more idiomatic

Reply:
Nice - thank you! :)